parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -41,7 +41,11 @@ installEtcd() {
             docker run --rm --entrypoint cat ${CONTAINER_IMAGE} /usr/local/bin/etcd > "$path/etcd"
             docker run --rm --entrypoint cat ${CONTAINER_IMAGE} /usr/local/bin/etcdctl > "$path/etcdctl"
         else
-            img unpack -o "$path" ${CONTAINER_IMAGE}
+            # img unpack requires a non-existent dirctory
+            tmpdir=/root/etcd${RANDOM}
```

Comment:
prefer `mktemp` ?

Reply:
So that's what I tried, but note the comment above - `img unpack` requires a *non-existent* directory. Once `mktemp` creates the directory, `img` will refuse to unpack to that path. Believe me I tried :) As I said earlier, I honestly don't know how this ever worked, and I couldn't find an `img` option to unpack to an existing directory.